25/11/21 (11:16:32) | ERROR: Exception while getting cpu frequency!
25/11/21 (11:16:32) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.67_2/libexec/lib/python3.10/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.67_2/libexec/lib/python3.10/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
25/11/21 (11:18:51) | ERROR: Exception while getting cpu frequency!
25/11/21 (11:18:51) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.67_2/libexec/lib/python3.10/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.67_2/libexec/lib/python3.10/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
25/11/21 (11:19:50) | ERROR: Exception while getting cpu frequency!
25/11/21 (11:19:50) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.67_2/libexec/lib/python3.10/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.67_2/libexec/lib/python3.10/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
15/12/21 (01:22:23) | ERROR: Exception while getting cpu frequency!
15/12/21 (01:22:23) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/homebrew/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/homebrew/Cellar/bpytop/1.0.67_2/libexec/lib/python3.10/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/homebrew/Cellar/bpytop/1.0.67_2/libexec/lib/python3.10/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
